content/components/action-bar.mdx

```diff
@@ -1,84 +1,89 @@
-    />
-    <Caption>Use only invisible icon buttons in an action bar.</Caption>
-  </Do>
-  <Dont>
+Note that depending on the variant used, the visual or label might not be visible.
+  </Text>
+  <Box>
     <img
-      src="https://user-images.githubusercontent.com/378023/193506488-44543352-f513-469e-8783-5d1cc7f44eaf.png"
+      src="https://user-images.githubusercontent.com/378023/198209541-7e647879-2096-46a3-b7bd-6d390ff72e41.png"
       role="presentation"
       width="456"
     />
-    <Caption>Don't use other variants or components in an action bar.</Caption>
-  </Dont>
-</DoDontContainer>
+  </Box>
+</Box>
 
 ### Dividers
 
-Dividers can be added to visually group related buttons.
-
-<DoDontContainer>
-  <Do>
-    <img
-      src="https://user-images.githubusercontent.com/378023/193506858-6542a6ba-4bac-400d-bba9-8601fbc032ed.png"
-      role="presentation"
-      width="456"
-    />
-    <Caption>Use a divider between buttons.</Caption>
-  </Do>
-  <Dont>
+<Box
+  as="figure"
+  display={["block", "block", "block", "block", "grid"]}
+  gridTemplateColumns="1fr 1fr"
+  gridGap={7}
+  m={0}
+  mb={3}
+>
+  <Text as="p" mt="0">
+    Use a divider to visually seperate items of the same type. No divider is needed between icons and buttons since they are already distinctive enough.
+  </Text>
+  <Box>
     <img
-      src="https://user-images.githubusercontent.com/378023/193506894-dc0a65a0-0f81-444a-aa8c-f3b0f74aecb5.png"
+      src="https://user-images.githubusercontent.com/378023/198211711-acca35ec-03d3-4ce1-9d02-05d65909e62f.png"
       role="presentation"
       width="456"
     />
-    <Caption>Don't use a divider at the beginning or end of the action bar.</Caption>
-  </Dont>
-</DoDontContainer>
+  </Box>
+</Box>
 
 ### Overflow menu
 
-When the buttons don't fit in the available space, an overflow button ("kebab" icon) is added at the end of the action bar signaling that there are more actions available. Clicking on the overflow button opens a menu with the remaining actions that didn't fit.
+When action bar items don't fit in the available space, an overflow button ("kebab" icon) is added at the end of the action bar signaling that there are more actions available. Clicking on the overflow button opens the overflow menu with the remaining actions that didn't fit.
 
 <img
-  src="https://user-images.githubusercontent.com/378023/193507064-4efe3f63-7b30-4656-8304-3dea3e3f1e03.png"
+  src="https://user-images.githubusercontent.com/378023/198549383-0e5f42fc-a456-481c-9c44-b9fd6202b70f.png"
   alt=""
   width="960"
 />
 
 #### Sorting
 
-Buttons that don't fit are added to the top of the menu. Meaning that the last button in the action bar will also be the last button when inside the menu:
+Actions that don't fit are added to the top of the menu. Meaning that the last action in the action bar will also be the last action when inside the menu. Make sure to add the most important actions to the front since they will be hidden last.
```

Comment:
```suggestion
Actions that don't fit are added to the top of the menu. Meaning that the last action in the action bar will also be the last action inside the menu. Make sure to add the most important actions to the front since they will be hidden last.
```